apps/ui/lib/components/Event/Message/DisplayReactions.tsx

```diff
@@ -0,0 +1,165 @@
+import React, { useState } from 'react';
+import emoji from 'node-emoji';
+import styled from 'styled-components';
+import { useSelector } from 'react-redux';
+import { Button, Txt } from 'rendition';
+import { useSetup } from '../../SetupProvider';
+import { selectors } from '../../../store';
+import { Contract } from 'autumndb';
+
+const Emoji = ({ symbol, count, ...rest }) => {
+	const em = React.useMemo(() => {
+		return emoji.emojify(symbol);
+	}, [symbol]);
+
+	return (
+		<Txt {...rest}>
+			{em}
+			<span>{count}</span>
+		</Txt>
+	);
+};
+
+const Wrapper = styled.section`
+	display: flex;
+	flex-direction: row;
+	align-content: space-between;
+	width: 200px;
+`;
+
+export const DisplayReactions = ({ message }: { message: Contract }) => {
+	const currentUser = useSelector(selectors.getCurrentUser());
+
+	const reactions = message.links?.['has attached element']?.filter(
+		(contract) => {
+			return contract.type === 'reaction@1.0.0';
+		},
+	);
+
+	const [groupReactions, setGroupReactions] = useState(
+		{} as { [key: string]: Contract[] },
+	);
+
+	const { sdk } = useSetup()!;
+	// TODO: Remove this workaround when this is resolved
+	// https://github.com/product-os/autumndb/issues/1333
+	React.useEffect(() => {
+		if (!reactions) return;
+		sdk
+			.query({
+				type: 'object',
+				$$links: {
+					'was created by': {
+						type: 'object',
+					},
+				},
+				properties: {
+					id: {
+						enum: reactions.map((reaction) => reaction.id),
+					},
+				},
+			})
+			.then((reactions) => {
+				const result = reactions.reduce((map, reaction) => {
+					const symbol = reaction.data.reaction as string;
+					map[symbol] = map[symbol] || [];
+					map[symbol].push(reaction.links?.['was created by']?.[0]!);
+					return map;
+				}, {} as { [key: string]: Contract[] });
+				setGroupReactions(result);
+			});
+	}, [reactions]);
+
+
+
+	// TODO: Enable this when this is resolved
```

Comment:
Can you please remove this commented-out code? Once the Autumndb issue is resolved we can review the code with fresh eyes. We're not really getting any benefit from leaving commented out code in this file.